rasa/core/policies/ensemble.py

```diff
@@ -67,10 +68,13 @@ def _check_for_important_policies(self) -> None:
         if not any(
             isinstance(policy, (MappingPolicy, RulePolicy)) for policy in self.policies
         ):
-            logger.info(
-                f"MappingPolicy not included in policy ensemble. Default intents "
-                f"'{USER_INTENT_RESTART} and {USER_INTENT_BACK} will not trigger "
-                f"actions '{ACTION_RESTART_NAME}' and '{ACTION_BACK_NAME}'."
+            rasa.shared.utils.io.raise_warning(
+                f"Neither '{RulePolicy.__name__}' nor '{MappingPolicy.__name__}' "
+                f"(deprecated) are included in the model's policy configuration. "
+                f"Default intents such as '{USER_INTENT_RESTART}' and "
+                f"'{USER_INTENT_BACK}' will not trigger actions "
+                f"'{ACTION_RESTART_NAME}' and '{ACTION_BACK_NAME}'.",
+                docs=DOCS_URL_ACTIONS,
```

Comment:
The URL points to https://rasa.com/docs/rasa/actions.
Maybe it should point to https://rasa.com/docs/rasa/default-actions ?